frontends/web/src/new_front/components/CreateSamples/CreateSamples/AnnotationInterfaces/Contexts/SelectBetweenImagesGenerative.tsx

```diff
@@ -104,20 +141,45 @@ const SelectBetweenImagesGenerative: FC<
       [field_names_for_the_model.original_prompt ?? "original_prompt"]: prompt,
     });
     setShowLoader(true);
-    const generatedImages = await post("/context/get_generative_contexts", {
-      type: generative_context.type,
-      artifacts: {
-        ...artifactsInput,
-        prompt: prompt,
-        user_id: user.id,
-      },
-    });
-    if (response.ok) {
-      setShowImages(generatedImages);
-      addElementToListInLocalStorage(artifactsInput.prompt, "promptHistory");
-      setPromptHistory(getListFromLocalStorage("promptHistory"));
-      setShowLoader(false);
-    }
+    const socket = new WebSocket(
```

Comment:
Is this  the  server sent event implementation?

Reply:
Yes